src/common/components/select-repository-list/index.js

```diff
@@ -125,6 +125,25 @@ export class SelectRepositoryListComponent extends Component {
     return renderedRepos;
   }
 
+  renderRepoAmount() {
+    const { ids, isFetching } = this.props.repositories;
+    const { selectedRepositories } = this.props;
+
+    if (isFetching && ids.length === 0) {
+      return (
+        <span>Calculating your total repositories&hellip;</span>
+      );
+    }
+
+    return (
+      <div>
```

Comment:
Is that additional `div` needed here? I think rendering it as inline element should be fine.